src/renderer/components/ft-list-video/ft-list-video.vue

```diff
@@ -206,6 +214,21 @@
           {{ $t('Search Listing.Label.Subtitles') }}
         </div>
       </div>
+
+      <ft-icon-button
+        v-if="(showDeArrowToggle && isHovered) || deArrowTogglePinned"
+        class="deArrowToggleButton"
+        :class="{alwaysVisible: deArrowTogglePinned}"
+        :icon="['fas', 'dot-circle']"
+        :color="deArrowChangedContent ? 'var(--primary-color)' : '#808080'"
```

Comment:
Please use the existing theme variables, don't hardcode colours, as it has to be usable on all themes. Additionally the ft-icon-button component already has a `theme` for changing the colours of it, please use that instead of introducing another way to do the same thing.

Reply:
I tried using the primary theme in `ft-icon-button` but the colours are reversed. I don't think I should create a new theme or change the current one. Not sure how to proceed.

**ft-icon-button.scss**
```
&.primary {
    background-color: var(--primary-color);
    color: var(--text-with-main-color);
```
**Current**
![image](https://github.com/user-attachments/assets/fc927d22-2bfe-4704-a975-543f7e526f18)
**If I reverse `color` with `background-color`**
![image](https://github.com/user-attachments/assets/ffcfc7c5-8ba7-409f-b4d2-2d4a8cd499fd)

Reply:
One fix would be to use the `--favorite-icon-color` as the "enabled" mode color, as that's one we already have set as a universal "other" state. You could also do `filter: grayscale(1);` for the disabled state and the primary/secondary icon color for the enabled state, which would match better with the existing DeArrow web UI.

Reply:
In the end I just ended up using `<font-awesome-icon>` and creating a custom css class for the button because the main and hover colours are different.